### PR TITLE
To override translation scope

### PR DESCRIPTION
**Pull request by odk211**
https://github.com/spree/spree_auth_devise/issues/340

### CODE REVIEW

File: lib/controllers/frontend/spree/user_confirmations_controller.rb [hunk 1]
```diff
@@ -11,4 +11,9 @@ class Spree::UserConfirmationsController < Devise::ConfirmationsController
   def after_confirmation_path_for(resource_name, resource)
     signed_in?(resource_name) ? signed_in_root_path(resource) : spree.login_path
   end
+
+  def translation_scope
+    'devise.confirmations'
```
Comment (odk211) [inline lib/controllers/frontend/spree/user_confirmations_controller.rb]: It seems not `devise.user_confirmations` but `devise.confirmations`.
https://github.com/spree/spree_auth_devise/blob/master/config/locales/en.yml#L7

**Comment by odk211** [comment]
`bin/bundle` output some error in circleci...

**Comment by odk211** [comment]
@damianlegawiec Can you check this PR?
I'm using spree 3.1-rc1 and find this issue.

**Comment by damianlegawiec** [comment]
@odk211 - sorry I've missed this PR - will check it tomorrow, but you're right we shouldn't break compatibility like this

**Comment by odk211** [comment]
@damianlegawiec ping

**Comment by odk211** [comment]
@damianlegawiec This PR is important for me.
I'm not sure why circleci failed, All test passes in my local.
Can you check it?

**Comment by damianlegawiec** [comment]
This took ages, but it's finally in master ;)

**Pull request closed without merging**